.babelrc

```diff
@@ -19,7 +19,7 @@
     }
   },
   "plugins": [
-    "@babel/plugin-syntax-dynamic-import",
```

Comment:
I think we can remove this dependency since we're not using it anymore, right?

Reply:
Removed